cli/hzn.go

```diff
@@ -658,6 +657,21 @@ Environment Variables:
 	utilVerifyPubKeyFile := utilVerifyCmd.Flag("public-key-file", msgPrinter.Sprintf("The path of public key file (that corresponds to the private key that was used to sign) to verify the signature of stdin.")).Short('K').Required().ExistingFile()
 	utilVerifySig := utilVerifyCmd.Flag("signature", msgPrinter.Sprintf("The supposed signature of stdin.")).Short('s').Required().String()
 
+	smCmd := app.Command("secretsmanager", msgPrinter.Sprintf("List and manage resources in the secrets manager. This CLI does not read and pull secret details; to do so, you must authenticate as an administrator and use the secrets manager CLI."))
```

Comment:
Please add .Alias("sm").Alias("secretsmanager") to the end of this command. That's how the short command names are created.
Also, please change the command description to: "... List and manage secrets in the secrets manager. NOTE: You must authenticate as an administrator to list secrets available to the entire organization. ..."

Reply:
addressed in recent changes